src/game/option/option_passport.c

```diff
@@ -122,16 +125,16 @@ static void Option_PassportFlipLeft(INVENTORY_ITEM *inv_item);
 
 void Option_PassportInit(void)
 {
-    g_SavegameRequester.item_texts = Memory_Alloc(
-        g_Config.maximum_save_slots * g_SavegameRequester.item_text_len);
-    m_SelectLevelRequester.item_texts = Memory_Alloc(
-        (g_GameFlow.level_count + 1) * m_SelectLevelRequester.item_text_len);
+    Requester_Init(&g_SavegameRequester, g_Config.maximum_save_slots);
+    Requester_Init(&m_SelectLevelRequester, g_GameFlow.level_count + 1);
+    Requester_Init(&m_NewGameRequester, MAX_GAME_MODES);
```

Comment:
Do I need to traverse each `item_blocked` and `item_texts` and initialize them to `NULL`? Should I also be doing that in `Requester_ClearTextstrings`?